docs/howto/overriding-templates.txt

```diff
@@ -0,0 +1,88 @@
+====================
+Overriding Templates
+====================
+
+Overriding from the project's templates directory
+=================================================
+
+To override templates from a package in ``django.contrib`` or from a third-party
+package, the preferred method is to create replacement templates in your
+project's templates directory.
+
+Let's say you're trying to override the templates for a third-party package
+called ``blog``, which provides the templates ``blog/post.html`` and
+``blog/list.html``. The relevant settings for your project would look like::
+
+    INSTALLED_APPS = [
+        ...,
+        blog,
+        ...,
+    ]
+
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [os.path.join(BASE_DIR, 'templates')],
+            'APP_DIRS': True,
+            ...
+        },
+    ]
+
+The :setting:`TEMPLATES` section here should already exist in your project
+settings, as it is normally created when you create a Django project. The
+setting that needs to be modified is :setting:`DIRS<TEMPLATES-DIRS>`.
+
+These settings assume you have a ``templates`` directory in the root of your
+project. To override the templates for the ``blog`` app, you'll create a folder
+in the ``templates`` directory, and add the correct files to that folder::
+
+    templates/
+        blog/
+            list.html
+            post.html
+
+When the configured template loader looks for the ``blog`` app templates, it
```

Comment:
This seems a bit wordy, perhaps something like "The template loader first looks for the blog app templates in the DIRS directory." would be enough.